modules/internal/ByteBufferHelpers.chpl

```diff
@@ -57,9 +57,14 @@ module ByteBufferHelpers {
     __primitive("chpl_comm_get", dest, src_loc_id, src_addr, len.safeCast(size_t));
   }
 
-  inline proc bufferAlloc(requestedSize) {
+  inline proc chpl_buf_size(requestedSize: int) {
```

Comment:
I think you wanted to make this look like `chpl_here_alloc`, but this kind of function naming makes it look too much like a runtime function (chpl_here_alloc is a runtime function). And "buffer size" sounds too general to me.

I'd rename this function to "getGoodAllocSize" and make it `private`. Chapel supports `private` module methods that prevent them to be used from the outside of the module. You just prepend `private` to make it `private inline proc getGoodAllocSize`

Reply:
> I'd rename this function to "getGoodAllocSize" and make it `private`. Chapel supports `private` module methods that prevent them to be used from the outside of the module. You just prepend `private` to make it `private inline proc getGoodAllocSize`

@e-kayrakli thanks for reviewing, I made the changes you suggested. 

> Thanks a lot! I hope to merge this soon after we unfreeze master.

Sure :-)